crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -359,7 +359,7 @@ impl CumulocityConverter {
                 let ancestors_external_ids =
                     self.entity_store.ancestors_external_ids(entity_topic_id)?;
 
-                if entity_topic_id.is_bridge_health_topic() {
+                if self.config.bridge_in_mapper && entity_topic_id.is_bridge_health_topic() {
                     return Ok(vec![]);
                 }
```

Comment:
Adding a comment would to understand what it means to return an empty vector of messages.

```suggestion
                if self.config.bridge_in_mapper && entity_topic_id.is_bridge_health_topic() {
                    // Ignore bridge health topics when using the mapper-provided bridge
                    return Ok(vec![]);
                }
```